app/assets/javascripts/discourse/package.json

```diff
@@ -14,7 +14,7 @@
     "build": "ember build",
     "start": "ember serve",
     "test": "ember test",
-    "postinstall": "yarn --silent --cwd .. patch-package"
+    "postinstall": "yarn --silent --cwd .. patch-package && yarn --cwd ../node_modules/ember-this-fallback install && yarn --cwd ../node_modules/ember-this-fallback build:ts"
```

Comment:
(hack to ensure `ts` is compiled. Eventually the compiled version will be published on npm, so this won't be necessary)